### PR TITLE
Adding a way to add asterisk automaticly to backform labels

### PR DESCRIPTION
**Pull request by patattackziliko**

### CODE REVIEW

File: src/backform.js [hunk 1]
```diff
@@ -116,6 +121,10 @@
           field: field,
           model: model
         });
+        if (form.showRequiredAsAsterisk && control.field.get('required')) {
```
Comment (martindrapeau) [inline src/backform.js]: Your switch should modify the rendering - not the actual control field. Instead, please pick out the label from the HTML, and add an *.

File: src/backform.js [hunk 2]
```diff
@@ -81,6 +81,7 @@
     fields: undefined,
     errorModel: undefined,
     tagName: 'form',
+    showRequiredAsAsterisk: false,
```
Comment (jdhorner) [inline src/backform.js]: This line isn't needed, really.

File: src/backform.js [hunk 3]
```diff
@@ -90,6 +91,10 @@
       this.fields = options.fields;
       this.model.errorModel = options.errorModel || this.model.errorModel || new Backbone.Model();
       this.controls = [];
+      
+      if(options.showRequiredAsAsterisk) {
+        this.showRequiredAsAsterisk = options.showRequiredAsAsterisk;
+      }
```
Comment (jdhorner) [inline src/backform.js]: Just the guts of the if should be all you need here. e.g.
`this.showRequiredAsAsterisk = options.showRequiredAsAsterisk;`
If they didn't give the option, it'll just be undefined.

File: src/backform.js (hunk 1 above)
Comment (patattackziliko) [inline src/backform.js]: I dont understand what you want me to do

Comment (martindrapeau) [inline src/backform.js]: Well my original idea was this:
```
control.$el.find('label').append(' *');
```

But looking at the code, it won't work for the BooleanControl. So nevermind, your solution is best.

File: src/backform.js [hunk 4]
```diff
@@ -116,7 +117,11 @@
           field: field,
           model: model
         });
-        $form.append(control.render().$el);
+        var $control = control.render().$el;
+        if(form.showRequiredAsAsterisk && control.field.get('required')) {
```
Comment (jdhorner) [inline src/backform.js]: `if (`

File: src/backform.js [hunk 5]
```diff
@@ -116,7 +117,11 @@
...
+          $control.find('label.' + Backform.controlLabelClassName).append(" *");
```
Comment (jdhorner) [inline src/backform.js]: `.append(' *');`

**Pull request merged**